### PR TITLE
Template out CONTRIBUTING.md and add instructions for E2E testing

### PR DESCRIPTION
**Pull request by fishkerez**

### CODE REVIEW

File: e2e/README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
-In order to run tests locally. need to increase docker daemon memory to 8G. (On mac with docker desktop)
+In order to run tests locally:
```
Comment (pbalogh-sa) [inline e2e/README.md:1]:
```suggestion
In order to run end-to-end tests locally:
```

File: e2e/README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+   sed -i 's/latest/v1.1/g' charts/kubeclarity/Chart.yaml
+   sed -i 's/latest/${{ github.sha }}/g' charts/kubeclarity/values.yaml
+   sed -i 's/Always/IfNotPresent/g' charts/kubeclarity/values.yaml
+   ```
+
+- make cli
+- mv ./cli/bin/cli ./e2e/kubeclarity-cli
+- make e2e (or run in debug like you will run a unit test)
+
+Note: Need to increase docker daemon memory to 8G. (On mac with docker desktop)
```
Comment (akpsgit) [inline e2e/README.md:16]: I think this note should be first

File: e2e/README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
-In order to run tests locally. need to increase docker daemon memory to 8G. (On mac with docker desktop)
+In order to run end-to-end tests locally:
+
```
Comment (akpsgit) [inline e2e/README.md:2]: I think that everything that is a command in here should be a copiable code snippet

File: e2e/README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
...
+   ```shell
+   sed -i 's/latest/v1.1/g' charts/kubeclarity/Chart.yaml
+   sed -i 's/latest/${{ github.sha }}/g' charts/kubeclarity/values.yaml
+   sed -i 's/Always/IfNotPresent/g' charts/kubeclarity/values.yaml
+   ```
+
+- make cli
+- mv ./cli/bin/cli ./e2e/kubeclarity-cli
+- make e2e (or run in debug like you will run a unit test)
```
Comment (akpsgit) [inline e2e/README.md:14]: not sure it's clear what "like you will run a unit test" means

File: e2e/README.md (hunk 3 above)
Comment (sambetts-cisco) [reply e2e/README.md:2]: agree, so things like `make cli` should be in code snippets

**Review by sambetts-cisco — CHANGES_REQUESTED** [review]
Lets revive this PR and get it merged because its super useful documentation. It looks pretty good but I agree with @akpsgit's comments about the code snippets.

We should also add a link somewhere in the main README that points to this document, probably in the Contributing section, or alternatively we need to add a CONTRIBUTING.md in the root of the project with a testing section that points to this.

**Comment by fishkerez** [comment]
Thanks for reviving it!

**Review by sambetts-cisco — APPROVED** [review]

**Review by pbalogh-sa — APPROVED** [review]

**Pull request merged**